src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/MethodologySummaryPage.tsx

```diff
@@ -80,6 +81,10 @@ const MethodologySummaryPage = () => {
               Edit summary
             </ButtonLink>
           )}
+
+          <MethodologyHelpAndSupportSection
+            owningPublication={methodology.owningPublication}
+          />
```

Comment:
This isn't what I'd expect the UI to look like at all. It seems like we're re-using the public frontend's contact section, which doesn't seem right as it's more tailored for the general public to read.

My expectation would have been that we'd display something similar to the summary list in `PublicationContactPage` i.e.

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/9917868/69f6d6ed-b21b-448e-bdfd-8156988806b6)

Reply:
As mentioned on Slack, I think that in the admin the contact details shouldn't be on the summary, they should be on the manage content page, and its preview version, same as how it is for releases.

Reply:
This has now been moved into the Manage Content section, but still uses the same `ContactUsSection` as per Amy's comment.